SU2_CFD/src/output_structure.cpp

```diff
@@ -4579,13 +4579,6 @@ void COutput::SetConvHistory_Body(ofstream *ConvHist_file,
 
           /*--- For specific applications, evaluate and plot the surface. ---*/
 
-          if (config[val_iZone]->GetnMarker_Analyze() != 0) {
```

Comment:
Why is this removed? Is SpecialOutput_Distortion not working for incompressible? (That's the only difference with the if statement below).